src/RuleSet/Sets/PERCS10Set.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\RuleSet\Sets;
+
+use PhpCsFixer\RuleSet\AbstractRuleSetDescription;
+
+/**
+ * @internal
+ *
+ * PER Coding Style v1.0.0.
+ *
+ * @link https://github.com/php-fig/per-coding-style/blob/1.0.0/spec.md
+ */
+final class PERCS10Set extends AbstractRuleSetDescription
```

Comment:
I think the name is confusing: is it PER CS 1.0 ou 10? I would override `getName()` to remove the ambiguity.